packages/@lwc/engine-core/src/shared/logger.ts

```diff
@@ -16,12 +16,6 @@ function log(method: 'warn' | 'error', message: string, vm?: VM) {
         msg = `${msg}\n${getComponentStack(vm)}`;
     }
 
-    if (process.env.NODE_ENV === 'test') {
-        /* eslint-disable-next-line no-console */
-        console[method](msg);
-        return;
-    }
-
```

Comment:
I couldn't find any explicit note about why this is here, but I traced it back to 2018: https://github.com/salesforce/lwc/pull/430#discussion_r196516442

If this causes problems in Jest, we'll probably find out soon enough from our Nucleus tests.

Reply:
There is an issue with Jest here. As far as I recall the goal was to reduce the warnings and errors verbosity in tests by avoiding printing the call stack. At this time there were a lot of warning false positives.

Reply:
OK, added it back with a comment.